Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKER_IMG = klakegg/hugo:0.80.0-ext
+DOCKER_IMG = klakegg/hugo:0.81.0-ext-asciidoctor
```

Comment:
What is the `-asciidoctor` suffix for?

Reply:
It's from the google/docsy-example repo. 
https://github.com/google/docsy-example/blob/1fa052f63b9768f4f245fe6b13f124bd12023e76/Dockerfile#L1

Ironically, the google/docsy-example won't build with docker currently because it's missing the `-ext` part (there's an open PR for it)

I include it because don't see a docker setup in the google/docsy repo, and am not sure if the `-asciidoctor` suffix is a req docsy, or just the example at this point.

Reply:
Works for me, thanks.